components/List.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  lists: { content: string }[];
+}
+export function List({ lists }: Props) {
+  const className = "grid grid-cols-1 gap-1 list-decimal";
```

Comment:
**praise:** We should add classnames directly to the element, not separate it from component.